v2.1/admin-ui-statements-page.md

```diff
@@ -0,0 +1,116 @@
+- The **Statements** page displays the details of the SQL statements executed only within a specified time interval. At the end of the specified time interval, the display is wiped clean, and you'll not see any statements on the **Statements** page until the next set of statements is executed. By default, the time interval is set to one hour; however, you can customize the interval using the [`diagnostics.reporting.interval`](cluster-settings.html#settings) cluster setting.
+
+## Understanding the Statements page
+
+### SQL statement fingerprint
+
+The **Statements** page displays the details of SQL statement fingerprints instead of individual SQL statements.
+
+A statement fingerprint is a grouping of similar SQL statements in their abstracted form by replacing the literal values with underscores (`_`). Grouping similar SQL statements as fingerprints helps you quickly identify the frequently executed SQL statements and their latencies.
+
+A statement fingerprint is generated when two or more statements are the same after any literal values in them (e.g. numbers and strings) are replaced with underscores. For example, the following statements have the same once their numbers have been replaced with underscores:
+
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (380, 11, 11098)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (192, 891, 20)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (784, 452, 78)`
+
+Thus, they can have the same fingerprint:
+
+`INSERT INTO new_order(product_id, customer_id, no_w_id) VALUES (_, _, _)`
+
+The following statements are different enough to not have the same fingerprint:
+
+- `INSERT INTO orders(product_id, customer_id, transaction_id) VALUES (380, 11, 11098)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (380, 11, 11098)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES ($1, 11, 11098)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES ($1, $2, 11098)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES ($1, $2, $3)`
+
+### Parameters
+
+The **Statements** page displays the time, count, and mean rows and latency for each statement fingerprint. By default, the statement fingerprints are sorted by time; however, you can sort the table by count, mean rows, and mean latency.
+
+The following details are provided for each statement fingerprint:
+
+Parameter | Description
+-----|------------
+Statement | The SQL statement or the fingerprint of similar SQL statements.<br><br>To view additional details of a statement fingerprint, click on the statement fingerprint in the **Statement** column to see the [**Statement Details** page](#statement-details-page).
+Time | The cumulative time taken to execute the SQL statement (or multiple statements having the same fingerprint).
```

Comment:
This description leaves me a little confused:

- If it's a fingerprint, does **Time** tell you the cumulative time for all of the statements grouped as the fingerprint, or for each individual statement? If the latter, is it an average?

Reply:
Since some of the other fields mention "average", I'm wondering if that's really the case here, too?

Reply:
We should rename this column to "total time" or "cumulative time". It's `average time * count` (both already columns in their own right).

Reply:
Also, I wouldn't say "the SQL statement (or multiple statements having the same fingerprint)" — it's always a fingerprint, it's just that sometimes there's only one statement with that fingerprint.

Reply:
I see what you mean. Maybe: 

> The cumulative time taken to execute the SQL statements represented by the fingerprint. 

@vilterp, when you say that average time is already a column, do you mean `Mean Latency`?

Reply:
Correct, `Mean Latency`. We're just doing users a favor by multiplying two columns already on the page.

Reply:
Yeah..I think the docs need to be more explicit in explaining that to the users. Will work on that in the follow-up PR.